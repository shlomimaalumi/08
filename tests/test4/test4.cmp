|RAM[5000|RAM[5001|RAM[5010|RAM[5011|
|      1 |      2 |      5 |     10 |
