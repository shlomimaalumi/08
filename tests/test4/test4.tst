load test4.asm,
output-file test4.out,
compare-to test4.cmp,
output-list RAM[5000]%D1.6.1 RAM[5001]%D1.6.1 RAM[5010]%D1.6.1 RAM[5011]%D1.6.1;

repeat 1000000 {
  ticktock;
}

output;
