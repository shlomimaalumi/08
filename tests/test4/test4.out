|RAM[5000|RAM[5001|RAM[5010|RAM[5011|
